lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -405,6 +405,8 @@ def _configure_node(
     ) -> None:
         node_context = get_node_context(node)
 
+        node_context.use_ignition = node_runbook.ignition
```

Comment:
Make sure the type is right.
`node_context.use_ignition = node_runbook.ignition is not None`